packages/web-components/api-extractor.json

```diff
@@ -18,11 +18,6 @@
     "enabled": true
   },
   "compiler": {
-    "tsconfigFilePath": "./tsconfig.lib.json",
-    "overrideTsconfig": {
-      "compilerOptions": {
-        "paths": {}
-      }
-    }
+    "tsconfigFilePath": "./tsconfig.api-extractor.json"
```

Comment:
why is this change necessary ? another exotic tsconfig === more confusion / not following repo patterns

Reply:
The API Extractor doesn't use the `tsconfigFilePath` if any `overrideTsconfig` fields are present. This was causing warnings "Subsequent property declarations must have the same type" (TS2717) with `@types/web` present.